src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+# Git Workflow
+The follow section outlines our Git workflow for Terra UI projects.
+
```

Comment:
@yuderekyu I think a table of contents in terra-ui will be broken because the hash router expects a hashed URL and markdown headers utilizes '#' as well. See https://github.com/cerner/terra-dev-site/issues/19

Reply:
correct, I wanted to but wasn't able to find a way to get it work with react-router.